tests/h/api/uri_test.py

```diff
@@ -30,134 +38,134 @@
     ("http%3A%2F%2Fexample.com%2Ffoo%2F", "http%3A%2F%2Fexample.com%2Ffoo%2F"),
 
     # Should leave already-normalized URLs as they are
```

Comment:
We should probably change the comment here -- as these aren't "already-normalised" URLs any more.